attachments.md

```diff
@@ -5,21 +5,21 @@
 
 ## Introduction
 
-Matroska supports storage of related files and data in the Attachments Top-Level Element. Attachments can be used to store related cover art, font files, transcripts, reports, error recovery files, picture or text-based annotations, copies of specifications, or other ancilliary files related to the Segment.
+Matroska supports storage of related files and data in the `Attachments Element` (a `Top-Level Element`). Attachments can be used to store related cover art, font files, transcripts, reports, error recovery files, picture or text-based annotations, copies of specifications, or other ancillary files related to the `Segment`.
 
 `Matroska Readers` MUST NOT execute files stored as Attachments.
 
 ## Cover Art
 
-This section defines a set of guidelines for the storage of cover art in Matroska files. A `Matroska Reader` MAY use embedded cover art to display a representation still-image depiction of the multimedia contents of the Matroska file.
+This section defines a set of guidelines for the storage of cover art in Matroska files. A `Matroska Reader` MAY use embedded cover art to display a representational still-image depiction of the multimedia contents of the Matroska file.
 
-Cover art SHOULD only use the JPEG and PNG picture formats.
+Cover art SHOULD only use JPEG and PNG formats.
 
-There can be 2 different covers for a movie/album. A portrait one (like a DVD case) and a landscape one (like a banner ad for example, looking better on a wide screen).
+There can be 2 different covers for a movie/album. A portrait style (e.g., a DVD case) and a landscape style (e.g., a wide banner ad).
```

Comment:
"two" instead of "2"? In German we usually use words for numbers up to and including 12. Is there something similar in English?